tests/Microsoft.DotNet.Docker.Tests/ImageScenarioVerifier.cs

```diff
@@ -104,10 +104,22 @@ private string CreateTestAppWithSdkImage(string appType)
 
             try
             {
+                string targetFramework;
+                // TODO: Until https://github.com/dotnet/aspnetcore/pull/19860 is fixed, 5.0 web projects need to
+                // continue to use the old TFM.
+                if (_imageData.Version.Major < 5 || appType == "web")
+                {
+                    targetFramework = $"netcoreapp{_imageData.Version}";
+                }
+                else
+                {
+                    targetFramework = $"net{_imageData.Version}";
+                }
+
                 _dockerHelper.Run(
                     image: _imageData.GetImage(DotNetImageType.SDK, _dockerHelper),
                     name: containerName,
-                    command: $"dotnet new {appType} --framework netcoreapp{_imageData.Version} --no-restore",
+                    command: $"dotnet new {appType} --framework netcoreapp{targetFramework} --no-restore",
```

Comment:
This doesn't look right.
```suggestion
                    command: $"dotnet new {appType} --framework {targetFramework} --no-restore",
```